pkg/component/observability/monitoring/prometheus/prometheus.go

```diff
@@ -109,11 +109,13 @@ func (p *prometheus) prometheus(takeOverOldPV bool, cortexConfigMap *corev1.Conf
 				Namespace: p.namespace,
 				Name:      p.values.Alerting.AlertmanagerName,
 				Port:      intstr.FromString(alertmanager.PortNameMetrics),
-				AlertRelabelConfigs: []monitoringv1.RelabelConfig{{
-					SourceLabels: []monitoringv1.LabelName{"ignoreAlerts"},
-					Regex:        `true`,
-					Action:       "drop",
-				}},
+				AlertRelabelConfigs: append(
+					[]monitoringv1.RelabelConfig{{
+						SourceLabels: []monitoringv1.LabelName{"ignoreAlerts"},
+						Regex:        `true`,
+						Action:       "drop",
+					}},
+					p.values.AdditionalAlertRelabelConfigs...),
```

Comment:
```suggestion
					p.values.AdditionalAlertRelabelConfigs...,
				),
```

Reply:
Oh sorry, I committed this but it seems I missed it again during rebasing.